module/metrics/transaction.go

```diff
@@ -21,6 +21,9 @@ type TransactionCollector struct {
 	timeToExecuted             prometheus.Summary
 	timeToFinalizedExecuted    prometheus.Summary
 	transactionSubmission      *prometheus.CounterVec
+	executeScriptDuration      *prometheus.CounterVec
+	transactionResultDuration  *prometheus.CounterVec
+	scriptSize                 *prometheus.HistogramVec
```

Comment:
This should also be a `prometheus.Histogram`

e.g. https://github.com/onflow/flow-go/blob/master/module/metrics/state_synchronization.go#L70-L76